src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml

```
@@ -0,0 +1,113 @@
+post:
+  summary: Copy project and packages instances.
+  description: Copy project and package instances.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: rev
+      schema:
+        type: string
+        enum:
+          - upload
+          - repository
+      example: repository
+    - in: query
+      name: opackage
+      schema:
+        type: string
+      description: Old package.
+    - in: query
+      name: oproject
+      schema:
+        type: string
+      description: Old project
+    - in: query
+      name: withbinaries
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Copy binaries also.
+    - in: query
+      name: makeolder
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Setting both makeolder and makeoriginolder will return an error back.
+    - in: query
+      name: makeoriginolder
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Setting both makeoriginolder and makeolder will return an error back.
+    - in: query
+      name: nodelay
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Set to '1' to defer the copy to a background job.
```

```suggestion
      description: Set to '1' to do the copying synchronously. By default the copying is done asynchronously.
```

background job is an implementation detail...